src/stores/simulation-store.ts

```diff
@@ -245,13 +274,22 @@ export const SimulationStore = types
     erupt(animate = false) {
       const rows = self.numRows;
       const cols = self.numCols;
-      const vX = self.volcanoX;
-      const vY = self.volcanoY;
+      const vLat = self.volcanoLat;
+      const vLng = self.volcanoLng;
+
+      self.windSpeed = self.stagingWindSpeed;
+      self.windDirection = self.stagingWindDirection;
+      self.colHeight = self.stagingColHeight;
+      self.mass = self.stagingMass;
+      self.vei = self.stagingVei;
+      self.particleSize = self.stagingParticleSize;
+
+      // This eruption data is no longer calculated correctly or used
```

Comment:
Do you think we are ever going to want to use this data again?  

If its only used for the old grid squares then maybe we can get rid of it.

Reply:
I removed a lot of the unused variables and put console warning messages in for the function calls until we get to changing the blockly code